contracts/token/ERC1155/ERC1155.sol

```diff
@@ -0,0 +1,234 @@
+pragma solidity ^0.5.0;
+
+import "./IERC1155.sol";
+import "./IERC1155TokenReceiver.sol";
+import "../../math/SafeMath.sol";
+import "../../utils/Address.sol";
+import "../../introspection/ERC165.sol";
+
+/**
+ * @title Standard ERC1155 token
+ *
+ * @dev Implementation of the basic standard multi-token.
+ * See https://eips.ethereum.org/EIPS/eip-1155
+ * Originally based on code by Enjin: https://github.com/enjin/erc-1155
+ */
+contract ERC1155 is ERC165, IERC1155
```

Comment:
Some of the contracts are missing, such as the one for the metadata and the one for the URI functions. Are those going to be implemented in another stage?

Reply:
Yeah. I did not seek to implement those contracts, since we don't know how these features really _should_ work because we don't use them, and didn't plan on using them. These should be added by somebody with more knowledge/use for those features.